docs/source/en/_toctree.yml

```diff
@@ -268,6 +268,8 @@
       title: Nyströmformer
     - local: model_doc/openai-gpt
       title: OpenAI GPT
+    - local: model_doc/opt
```

Comment:
The model will also need to be added to the main README.

Reply:
I am not really I know where I should add it, could you help me out?

Reply:
on it